samples/server/petstore/php-slim/pom.xml

```diff
@@ -0,0 +1,43 @@
+<project>
+    <modelVersion>4.0.0</modelVersion>
+    <groupId>org.opoenapitools</groupId>
+    <artifactId>SlimPetstoreServerTests</artifactId>
+    <packaging>pom</packaging>
+    <version>1.0-SNAPSHOT</version>
+    <name>Slim Petstore Server</name>
+    <build>
+        <plugins>
+            <plugin>
+                <artifactId>maven-dependency-plugin</artifactId>
+                <executions>
+                    <execution>
+                        <phase>package</phase>
+                        <goals>
+                            <goal>copy-dependencies</goal>
+                        </goals>
+                        <configuration>
+                            <outputDirectory>${project.build.directory}</outputDirectory>
+                        </configuration>
+                    </execution>
+                </executions>
+            </plugin>
+            <plugin>
+                <groupId>org.codehaus.mojo</groupId>
+                <artifactId>exec-maven-plugin</artifactId>
+                <version>1.2.1</version>
+                <executions>
+                    <execution>
+                        <id>syntax-check</id>
+                        <phase>integration-test</phase>
+                        <goals>
+                            <goal>exec</goal>
+                        </goals>
+                        <configuration>
+			    <executable>./php_syntax_checker.bash</executable>
```

Comment:
please replace tabs with spaces.

Reply:
👍 Yup, noticed that as well. Done via https://github.com/OpenAPITools/openapi-generator/pull/723/commits/9185150c0a0122bfe5ff1af2a9c759c41212d1da

Reply:
Done via f83df38. Also updated vim setting to use spaces instead of tabs.